awx_collection/README.md

```diff
@@ -32,7 +32,7 @@ Installing the `tar.gz` involves no special instructions.
 ## Running
 
 Non-deprecated modules in this collection have no Python requirements, but
-may require the official [AWX CLI](https://docs.ansible.com/ansible-tower/latest/html/towercli/index.html)
+may require the official [AWX CLI](https://docs.ansible.com/ansible-controller/latest/html/controllercli/index.html)
```

Comment:
do we plan to change urls?

Reply:
We should (or move the docs to access.redhat.c0m)

Reply:
Though in this case it would be the automation-controller, under the Automation Platform. It is *not* called the "ansible controller"